docs/UserGuide.md

```diff
@@ -172,47 +174,52 @@ You may activate the command box by simply pressing the spacebar on your keyboar
 Similarly, you may press the `ESC` key on your keyboard to exit out of the command box and hide the results display.
 </div>
 
-[Back to Top](#sectresbook)
+[Back to Table of Contents](#table-of-contents)
 
 ### People Panel
 
-This panel contains all the club and organisation members you have registered in this book. They are laid out horizontally. Although there is no need to physically control list, the user may still scroll the list with their mouse, or by pressing on the horizontal scroll bar and scrolling.
+The People Panel contains all the club and organisation members you have registered in this book. They are laid out horizontally. You can scroll the list by hovering your mouse over the People Panel and scrolling the mouse-wheel, or by clicking on and dragging the horizontal scroll bar to scroll.
+
+Each card represents a person and displays their name, phone number and total present loan amount. The loan amount may be positive to indicate an amount owed by the person, or negative to indicate an amount due to be paid to the person.
 
-Each card representing the information of a person displays the name, phone number and total present loan amount, which may be positive to indicate an amount owed by the person, or negative to indicate an amount due to be paid to the person.
+<p align="center">
+  <img src="images/TypicalPersonCard.png" width="125">
```

Comment:
Were you also going to add the notes card in the Notes Panel description below?

Reply:
Updated